rpcs3/Emu/Memory/vm.cpp

```diff
@@ -112,15 +119,15 @@ namespace vm
 					reservation_notifier(addr).notify_all();
 				}
 
+				if (cpu && !had_wait && cpu->test_stopped())
+				{
+					//
```

Comment:
?

Reply:
Prevents warning of test_stopped() (discard return value)

Reply:
Ah, okay. Could've written that explanation in the comment